tests/risks/test_low_birth_weight_and_short_gestation.py

```diff
@@ -56,11 +56,15 @@ def test_lbwsg_risk_effect_rr_pipeline(
     # Create exposure with matching demograph index as age_bins
     age_bins = make_age_bins()
     agees = age_bins.drop(columns="age_group_name")
-    exposure_data = make_categorical_exposure_data(agees)
+    # Have to match age bins and rr data to make age intervals
+    rr_data = make_categorical_data(agees)
+    # Exposure data used for risk component
+    exposure = make_categorical_data(agees)
```

Comment:
Is this supposed to be the same as rr_data? I'm confused

Reply:
It is not suppose to be but it is for this test